agm.html

```diff
@@ -0,0 +1,78 @@
+<!--#set var="title" value="AGM 2021" -->
+<!--#include virtual="inc/header.html" -->
+<h2>SRCF AGM 2021</h2>
+
+<div class="column">
+  <div class="row">
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="card">
+        <div class="card-body">
+          <h4 class="card-title">Meeting details</h4>
+          <p>The 2021 Annual General Meeting of the Student-Run Computing Facility
+            will be held during the final week of Lent term.</p>
+          <p>The meeting will commence at 14:00 on Saturday 20 March 2021.</p>
+          <p>You will be able to join the AGM via Timeout via <a href="https://timeout.srcf.net/agm">https://timeout.srcf.net/agm</a>.</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="card w-100">
+        <div class="card-body">
+          <h4 class="card-title">Standing Agenda Items</h4>
+          <ul>
+            <li>Executive annual reports
+              <ul>
+                <li>Chair's report</li>
+                <li>Treasurer's report</li>
+                <li>Sysadmins' report</li>
+              </ul>
+            </li>
+            <li>Consideration of received motions, if any</li>
+            <li>Election procedure
+              <ul>
+                <li>Election candidates' speeches</li>
+                <li>Voting (procedure TBD)</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-body">
+      <h4 class="card-title">Call for Submissions</h4>
+      <p>If you wish to stand for election to the committee, please read the dedicated section below.</p>
+      <p>If you wish to submit a motion for the AGM, please contact the Secretary at least three days in advance of the meeting date with details of the motion.  Examples of types of motion are resolutions to adopt a policy, mandates to the SRCF committee, or constitutional amendments.</p>
+    </div>
+  </div>
+
+  <div class="card">
+    <div class="card-body">
+      <h4 class="card-title">Committee Elections</h4>
+      <p>The AGM will include the election of the SRCF Committee for 2021-22.</p>
+      <p>We strongly encourage anyone interested, regardless of prior involvement with the SRCF, to nominate themselves for a committee position. All committee positions have a low time commitment and are accessible for those with any amount of experience. It's an incredibly valuable experience, and you'll be contributing to a society that is so crucial to hundreds of our fellow societies.</p>
```

Comment:
I realise these words are probably based on mine from a year ago, although I've just realised I make mention of all the societies we help, but none of the individuals!

In general I think this line can be more strong of a 'call to action' -- e.g. "Regardless of any prior involvement in the SRCF, we encourage you to stand for a committee position if you care about championing free and accessible computing for everyone at Cambridge."